terra_layer/tests/test_views.py

```diff
@@ -309,7 +309,7 @@ def test_layer_view_with_custom_style(self):
                 },
             ],
             minisheet_enable=True,
-            popup_enable=True,
+            popup_config={"enabled": True},
```

Comment:
enable vs enabled ? Check consistency.

Reply:
I changed it. It is now `enable` instead of `enabled`